browser-sign-in/ios/browserSignIn/AppDelegate.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Change copyright in all .h and .m files to Okta copyright

Reply:
Changed